Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN chmod a+rwX /files
 RUN set -x \
 #  && apt-get update \
   && apt-get install -y inotify-tools \
-  && wget -O /files/filebot.deb 'https://app.filebot.net/download.php?type=deb&arch=amd64&version=4.7.2' \
+  && wget -O /files/filebot.deb 'https://app.filebot.net/download.php?type=deb&arch=amd64&version=4.7.7' \
```

Comment:
I thought that it would run that on each client. If it's just run once when the container is published then this is indeed much less of an issue.